schemas/common/itemselector.schema.json

```diff
@@ -0,0 +1,72 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/itemselector",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Item Selector",
+  "meta:extensible": true,
```

Comment:
Should it be extensible? It looks like a low-level plumbing schema where we will be unlikely to be able to handle extensions in any implementation.